### ISSUE TITLE
Latest Release and Call recorder bugs

### ISSUE THREAD

**Issue by lirane1** [GrapheneOS/os-issue-tracker#4307]
The current call recorders has several bugs.

1. The call recorder stops recording when you add someone to the call (Using conference call)
2. When someone call and you do not accept the call, the call recording also stops.
3. There is no auto call recording for all calls, also I would really like to have a call recording exception list.

**Comment by marekm72** [comment]
Yes, please do implement the 3rd one - asking here because https://github.com/GrapheneOS/os-issue-tracker/issues/4740 is locked.  Yes I know what I'm doing, would find it very useful, and happen to live in a country where it is 100% legal to record my own conversations.  Just my +1.  Thanks, and please do keep up the good work!

**Comment by inthewaves** [comment]
> When someone call and you do not accept the call, the call recording also stops.

This was fixed by https://github.com/GrapheneOS/platform_packages_apps_Dialer/pull/56

> The call recorder stops recording when you add someone to the call (Using conference call)

This will be an intentional design choice for now (e.g. Google Dialer doesn't even allow recordings of conference calls). As another party would be added, users should actively turn call recording back on for conferences to acknowledge the new parties that would be recorded. Note that if you yourself are added to conference, the system might have no way of knowing that in some cases...
